packages/react/src/components/UserProvider.tsx

```diff
@@ -31,6 +30,7 @@ export interface Props {
   profileUrl?: string;
   user?: User;
   fetcher?: UserFetcher;
+  pathname: any;
```

Comment:
```suggestion
  pathname: string;
```

should this be `string`?

Reply:
Yeah I think you are correct here.